rush-plugins/rush-resolver-cache-plugin/src/computeResolverCacheFromLockfileAsync.ts

```diff
@@ -136,6 +136,15 @@ export interface IComputeResolverCacheFromLockfileOptions {
   ) => Promise<void>;
 }
 
+/**
+ * Copied from `@rushstack/node-core-library/src/Path.ts` to avoid expensive dependency
+ * @param path - Path using backslashes as path separators
+ * @returns The same string using forward slashes as path separators
+ */
+function convertToSlashes(path: string): string {
```

Comment:
Why not import the method from node core library? Trying to avoid the overhead?

Reply:
Yeah, imports from node-core-library are really expensive. It's literally in the code comment. :).

Reply:
Ah... I didn't read the comment cause I knew what it was doing :P